functions/package.json

```diff
@@ -43,6 +43,7 @@
     "@types/ioredis": "^4.27.1",
     "@types/jest": "^26.0.24",
     "@types/node": "^14.18.3",
+    "@types/node-fetch": "^2.6.4",
```

Comment:
I think you need to have the Algolia package in the function package.json for it to work.